content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,305 @@
+
+
+To link your DDR and primary orgs, run these commands replacing the placeholders for their values:
+
+<div class="alert alert-warning"> For security reasons, Datadog is unable to link the orgs on your behalf. </div>
+
+
+```shell
+export PRIMARY_DD_API_KEY=<PRIMARY_ORG_API_KEY>
+export PRIMARY_DD_APP_KEY=<PRIMARY_ORG_APP_KEY>
+export PRIMARY_DD_API_URL=<PRIMARY_ORG_API_SITE>
+
+export DDR_ORG_ID=<DDR_ORG_PUBLIC_ID>
+export PRIMARY_ORG_ID=<PRIMARY_ORG_PUBLIC_ID>
+export USER_EMAIL=<USER_EMAIL>
+export CONNECTION='{"data":{"id":"'${PRIMARY_ORG_ID}'","type":"hamr_org_connections","attributes":{"TargetOrgUuid":"'${DDR_ORG_ID}'","HamrStatus":1,"ModifiedBy":"'${USER_EMAIL}'", "IsPrimary":true}}}'
+
+curl -v -H "Content-Type: application/json" -H 
+"dd-api-key:${PRIMARY_DD_API_KEY}" -H 
+"dd-application-key:${PRIMARY_DD_APP_KEY}" --data "${CONNECTION}" --request POST ${PRIMARY_DD_API_URL}/api/v2/hamr
+```
+{{% /collapse-content %}}
+
+<br>
+
+<!-- ------------------------------- -->
+
+
+### 2. Set up access, integrations, syncing, and agents
+
+{{% collapse-content title="Create your Datadog API and App key for syncing" level="h5" %}}
+In DDR Datadog org, create a set of `API key` **and** `App key`. These are useful for copying dashboards and monitors between Datadog sites. 
+
+<div class="alert alert-info">
+Datadog can help copy the API key signatures for your Agents to the DDR back-up account. Contact your <a href="mailto:success@datadoghq.com">Customer Success Manager</a> for any questions regarding this.
+</div>
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Configure Single Sign On for the Datadog App" level="h5" %}}
+Go to your [Organization Settings][1] to configure [SAML][9] or Google Login for your users. 
+
+**Datadog recommends using Single Sign On (SSO)** to enable all your users to seamlessly login to your Disaster Recovery organization during an outage.
+
+You must invite each of your users to your Disaster Recovery organization and give them appropriate roles and permissions. Alternatively, to streamline this operation, you can use [Just-in-Time provisioning with SAML][2].
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" Set up your cloud integrations (AWS, Azure, Google Cloud)" level="h5" %}}
+
+Your cloud integrations must be configured in both primary and DDR organizations. Because these integrations only run in one data center at a time, **the integrations must run only in the primary data center.**
+
+During testing, integration telemetry is spread over both organizations and cancelling a failover testing returns the integrations to running in the Primary data center.
+
+During an integration failover, integrations runs only in the DDR data center.
+
+
+{{% /collapse-content %}}
+
+
+
+{{% collapse-content title="Set up Resources syncing and scheduler" level="h5" id="using-sync-cli-tool" %}}
```

Comment:
```suggestion
{{% collapse-content title="Set up resource syncing on a schedule" level="h5" id="using-sync-cli-tool" %}}
```
Just playing around here a bit to try to make wording easier to understand, in case someone thought, say, "Resources syncing" might be the name of a feature.